tests/functional/plugins/test_sqlite.py

```diff
@@ -0,0 +1,67 @@
+import os
+import json
+import pandas
+import pytest
+import sqlite3
+from dbt.tests.util import (
+    check_relations_equal,
+run_dbt,
+)
+
+model_sql = """
+    {{ config(materialized='incremental', database='satest') }}
+    select * from satest.tt1
+"""
+
+
+class TestSQLitePlugin:
+
+    @pytest.fixture(scope="class")
+    def sqlite_test_db(self):
+        path = '/tmp/satest.db'
+        os.unlink(path)
+        db = sqlite3.connect(path)
+        cursor = db.cursor()
+        cursor.execute("CREATE TABLE tt1 (id int, name text)")
+        cursor.execute("INSERT INTO tt1 VALUES (1, 'John Doe')")
+        cursor.execute("INSERT INTO tt1 VALUES (2, 'Jane Smith')")
+        cursor.execute("CREATE TABLE test_table2 (a int, b int, c int)")
+        cursor.execute("INSERT INTO test_table2 VALUES (1, 2, 3), (4, 5, 6)")
+        cursor.close()
+        db.commit()
+        db.close()
+
+        yield path
+
+    @pytest.fixture(scope="class")
+    def profiles_config_update(self, dbt_profile_target, sqlite_test_db):
+        return {
+            "test": {
+                "outputs": {
+                    "dev": {
+                        "type": "duckdb",
+                        "path": dbt_profile_target.get("path", ":memory:"),
+                        "attach": [
+                           {'path': sqlite_test_db}
```

Comment:
Doesn't this need to have the `sqlite` type included? I thought the default assumed a DuckDB database was being attached

Reply:
They show this approach in the `sqlite_scanner` [docs](https://github.com/duckdb/sqlite_scanner#reading-data-from-sqlite)

I believe if the file has the `.db` extension it is assumed to be sqlite